metpy/calc/thermo.py

```diff
@@ -1038,3 +1038,99 @@ def most_unstable_parcel(p, temperature, dewpoint, heights=None,
     theta_e = equivalent_potential_temperature(p_layer, T_layer)
     max_idx = np.argmax(theta_e)
     return p_layer[max_idx], T_layer[max_idx], Td_layer[max_idx]
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]')
+def mixed_parcel(p, temperature, dewpoint, parcel_start_pressure=None, bottom=None,
+                 heights=None, depth=100 * units.hPa, interpolate=True):
+    r"""Calculate the properties of a parcel mixed from a layer.
+
+    Determines the properties of an air parcel that is the result of complete mixing of a
+    given atmospheric layer.
+
+    Parameters
+    ----------
+    p : array-like
+        Atmospheric pressure profile
+    temperature : array-like
+        Atmospheric temperature profile
+    dewpoint : array-like
+        Atmospheric dewpoint profile
+    parcel_start_pressure : `pint.Quantity`
```

Comment:
From here on down, all the arguments are optional, and should be noted as such in the docs (along with the default values).